index.html

```diff
@@ -0,0 +1,179 @@
+                                <i class="fa fa-university"></i>
+                                <h3>Exhibition</h3>
+                            </div>
+                            <div class="col-6 col-md-12">
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos asperiores</p>
+                            </div>
+                        </div>
+
+                    </div>
+                    <div class="col-md margin-bg">
+                        <div class="program-stats bg-transparen">
+                            <div class="program-icon col-6 col-md-12">
+                                <i class="fa fa-cutlery"></i>
+                                <h3>Dinner</h3>
+                            </div>
+                            <div class="col-6 col-md-12">
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos asperiores </p>
+                            </div>
+
+                        </div>
+
+                    </div>
+                    <div class="col-md margin-bg">
+                        <div class="program-stats bg-transparen">
+                            <div class="program-icon col-6 col-md-12">
+                                <i class="fa fa-users"></i>
+                                <h3>Workshop</h3>
+                            </div>
+                            <div class="col-6 col-md-12">
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos asperiores</p>
+                            </div>
+
+                        </div>
+
+                    </div>
+
+                </div>
+                <p class="see-more">see the whole program</p>
+                <button type="button" class="join-btn">Join Child-support program</button>
+            </div>
+
+            </section>
+
+            <section class="speakers">
+            <div class="container">
+                <div class="speaker-heading">
+                    <h3>Guest Speakers</h3>
+                </div>
+
+                <div class="row speaker-info">
+
+                </div>
+
+            </div>
+
+        </section>
+
+        <footer>
+        <div class="footer-icon">
+            <div class="container">
+                <h2>partner</h2>
+                <div class="d-flex justify-content-center ">
+                    <ul class="d-flex justify-content-center flex-wrap list-unstyled align-items-center">
+                        <li><img src="./images/kakao1.png" alt="kakao1"></li>
+                        <li><img src="./images/google2.png" alt="google2"></li>
+                        <li><img src="./images/navier1.png" alt="navier1"></li>
+                        <li><img src="./images/mozilla.png" alt="mozilla"></li>
+                        <li><img src="./images/airbnd1.png" alt="airbnd1"></li>
+                    </ul>
+                </div>
+
+            </div>
+        </div>
+
```

Comment:
- [ ] [Optional] The background in this part is making the content of the page to be invisible, hence it's worsening the user experience, would you fix that before you merge this branch? 
![Screenshot from 2021-11-26 12-36-34](https://user-images.githubusercontent.com/56886064/143568020-82ecc2e8-2c2b-462d-918f-5f8e5887e740.png)